content/_code-samples/key-derivation/key-derivation.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python3
+
+################################################################################
+# XRPL Key Derivation Code
+# Author: rome@ripple.com
+# Copyright Ripple 2019
+# This sample code is provided as a reference for educational purposes. It is
+# not optimized for speed or for security. Use this code at your own risk and
+# exercise due caution before using it with real money or infrastructure.
+# This file is provided under the MIT license along with the rest of the
+# XRP Ledger Dev Portal docs and sample code:
+# https://github.com/ripple/xrpl-dev-portal/blob/master/LICENSE
+# Some of its dependencies are released under other licenses or are adapted
+# from public domain code. See their respective files for details.
+################################################################################
+
+import argparse
+import sys
+from hashlib import sha512
+
+if sys.version_info[0] < 3:
+    sys.exit("Python 3+ required")
+elif sys.version_info.minor < 6:
+    from random import SystemRandom
+    randbits = SystemRandom().getrandbits
+else:
+    from secrets import randbits
+
+from fastecdsa import keys, curve
+
+import ed25519
+import RFC1751
+import base58.base58 as base58
```

Comment:
I get an error err that `base58.base58` is not a package (if I change this to just `base58` it works OK).